great_expectations/expectations/expectation.py

```diff
@@ -282,7 +282,7 @@ class Config:
 
     id: Union[str, None] = None
     meta: Union[dict, None] = None
-    notes: Union[str, None] = None
+    notes: Union[str, List[str], None] = None
```

Comment:
Meta notes support strings and lists of strings - we should make this consistent